cmd/thor/flags.go

```diff
@@ -64,10 +63,15 @@ var (
 		Usage: "enables API requests logging",
 	}
 	verbosityFlag = cli.IntFlag{
-		Name:  "verbosity",
-		Value: int(log15.LvlInfo),
+		Name: "verbosity",
+		// Legacy for log15.LvlDebug
```

Comment:
just to double check if these are in docs :)

Reply:
Ya its there

```
| `--verbosity`               | Log verbosity (0-9) (default: 3)                                                            |
```